packages/test/test-gc-sweep-tests/src/mockConfigProvider.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ConfigTypes, IConfigProviderBase } from "@fluidframework/telemetry-utils";
+
+export const mockConfigProvider = ((settings: Record<string, ConfigTypes>): IConfigProviderBase => {
```

Comment:
Could be useful to move this to some other test utils package.

Reply:
Good point, it may interfere with your work, how about I do it as a separate task?